src/modes.cpp

```diff
@@ -0,0 +1,206 @@
+#include "main.hpp"
+
+/**
+ * @brief Initializes the mode of the WaypointNode.
+ *
+ * This function is responsible for initializing the mode of the WaypointNode.
+ * It checks if a command is specified and performs the necessary actions based
+ * on the command. If a pre_wait_time is specified, it sets the node state to
+ * pre_wait_time and initializes a wait timer. If no pre_wait_time is specified,
+ * it skips the pre_wait_time state and sets the node state to
+ * reach_cruise_height.
+ */
+void WaypointNode::mode_init() {
+    if (get_state_first_loop()) {
+        // Check if cmd is specified
+        if (!cmd.values_set) {
+            RCLCPP_FATAL(this->get_logger(),
+                         "WaypointNode::mode_init: Node was "
```

Comment:
"specifing" should be "specifying"

Reply:
Fixed in https://github.com/THI-Drone/waypoint_package/pull/3/commits/5a49068ac86cdab12bbed671d77177ca7fee6d9a